tailwind.config.js

```diff
@@ -1,6 +1,7 @@
 /** @type {import('tailwindcss').Config} */
 export default {
   content: ["./index.html", "./src/**/*.{js,ts,jsx,tsx}"],
+  darkMode: "class",
```

Comment:
What does this line mean?

Reply:
That line is what makes the magic! That line is necessary for Tailwin to understand the change to dark mode.